environments/staging/public.yml

```diff
@@ -152,6 +152,7 @@ localsettings:
   RATE_LIMIT_SUBMISSIONS: yes
   SES_CONFIGURATION_SET: staging-email-events
   SNS_EMAIL_EVENT_SECRET: "{{ SNS_EMAIL_EVENT_SECRET }}"
+  RECAPTCHA_PUBLIC_KEY: '6LeaE-sdAAAAAJm75fPLW9j0TBuX5dMUEJ14rQzw'
```

Comment:
Even though it's not strictly secret (and is visible in browsers), we have a convention of storing both the public and private key of a key pair (or analogously both the username and the password for a set of credentials) as "secrets". Part of it is that they logically go together and they change together, so it's more convenient if when they change you can change them the same way. Part of it is that keys like this _look_ like secrets, which can get people used to putting things that _look_ like secrets in `public.yml`, which could down the road easily lead to a mistake where we put an actual secret there.

So my in short my suggestion is that you move this into `secrets.yml` like the other one.